### PR TITLE
for working testing of input fields(don't merge)

### PR DESCRIPTION
**Pull request by maze-runnar**
@saurabh28march @SARTHAK1SINGH test the changes!!

### CODE REVIEW

**Comment by SARTHAK1SINGH** [comment]
is navbar working ??

**Comment by maze-runnar** [comment]
@SARTHAK1SINGH I have added something new to resolve problem. Now it will work.

File: templates/index.html [hunk 1]
```diff
@@ -18,26 +18,55 @@
...
+	{{answer}}
+<h3>Your mentioned Disease can be cured by following drugs: </h3>
+    {% for i in d %}
+      <li>  {{i}} </li>
+    {% endfor %}
+<h3>Your mentioned Medicines can cure the following disease: </h3>
+    {% for i in c %}
+      <li>  {{i}} </li>
+    {% endfor %}
+
```
Comment (saurabh28march) [inline templates/index.html:69]: add link to extrenal javascript file

File: templates/index.html [hunk 2]
```diff
@@ -18,26 +18,55 @@
...
         </div>
     </div>
-
+	{{answer}}
+<h3>Your mentioned Disease can be cured by following drugs: </h3>
+    {% for i in d %}
+      <li>  {{i}} </li>
+    {% endfor %}
+<h3>Your mentioned Medicines can cure the following disease: </h3>
+    {% for i in c %}
```
Comment (saurabh28march) [inline templates/index.html:66]: use other variable names

**Pull request closed without merging**